src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+@use '../_variables/variables' as *;
+@use '../mixins/mixins.scss' as *;
+
+.card {
+  width: $CardWidth;
+  padding: $CardWidth * 0.01 * 16 $CardWidth * 0.01 * 8 $CardWidth * 0.01 * 8;
+  border: 1px solid $ColorCardBorder;
```

Comment:
The variable `$ColorCardBorder` should be written in lowercase letters with hyphens instead of camelCase, as per the code style rules.